api/migrations/20181119221918_remove-activity-key-personnel.js

```diff
@@ -0,0 +1,6 @@
+exports.up = async knex =>
+  knex.schema.alterTable('activity_state_peronnel', table => {
```

Comment:
typo?

Reply:
Yes, but not here.  😆  That table name has been wrong since the beginning.  In my migration squash coming up, I'm planning to fix it.  🤞